README.md

```diff
@@ -342,14 +342,17 @@ turned off but may be activated by setting the environment variable
 ISAR_MQTT_ENABLED = true
 ```
 
-The connection to the broker will be determined by the following configuration values which may all be overwritten
-through the environment.
+The connection to the broker will be determined by the following configuration values specified in ```settings.env```
```

Comment:
I think the previous text is better here, as the variables have default values defined in the [settings.py](https://github.com/equinor/isar/tree/main/src/isar/config/settings.py) file

Reply:
In practice default values are specified in settings.env since the environment overwrite settings.py, but I will modify to include both in text.